proxy/http/http.go

```diff
@@ -233,11 +259,13 @@ func (r *Response) ReadFrom(discardBody bool, reader *bufio.Reader) error {
 	if err := util.WriteWithValidation(r.writer, respLineBytes); err != nil {
 		return util.ErrWrapper(err, "fail to write start line of response")
 	}
+	r.writeSize += len(respLineBytes)
 
 	//read & write the headers
 	var hijackerBodyWriter io.Writer
 	if err := copyHeader(&r.header, reader, r.writer,
 		func(rawHeader []byte) {
+			r.writeSize += len(rawHeader)
```

Comment:
not exactly the returned header traffic, cause the header has been translated and modified